onnxconverter_common/onnx2py.py

```diff
@@ -177,7 +177,8 @@ def convert_tensor(tensor):
         np_data = np_data.astype(np.str)
     os.makedirs(const_dir, exist_ok=True)
     name = "const" + str(const_counter)
-    if tensor.name:
+    if tensor.name and len(tensor.name) < 100:
```

Comment:
how does number '100' come from?

Reply:
260 is the max path length, but 100 just seemed like a reasonable value less than it since the tensor name only forms part of the path.  I could check the entire path length, but the model could be moved around later so I think it is best to choose a reasonably safe length (100) and use that.